src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/PpudOffender.kt

```diff
@@ -27,6 +30,9 @@ data class PpudSentence(
   val mappaLevel: String,
   val licenceExpiryDate: String?,
   val offence: PpudOffence?,
+  val releaseDate: String?,
+  // deprecated
+  @JsonIgnore
   val releases: List<PpudRelease>?,
```

Comment:
Why keep the field? Are we still using it somewhere?

Reply:
Discussed: kept because existing recommendations with releases will fail on retrieval otherwise.